src/components/Calculator.js

```diff
@@ -31,14 +31,18 @@ function Calculator() {
   };
 
   return (
+
     <div
       style={{
         display: 'flex',
         flexDirection: 'column',
         width: 400,
         height: 'auto',
+        paddingLeft: 700,
       }}
     >
+      {' '}
+      <p>Lets do some math!</p>
       <div
         style={{
           display: 'flex',
```

Comment:
- [ ] One of the requirements is to style your page to look identical to that of the wireframe provided. Kindly style the `Calculator` component to look like the wireframe below:

![image](https://github.com/Kidd254/Math-Magicians-React-App/assets/77106962/eea97c87-8d9b-4241-bce5-a9a4f829c216)

### Your Design

![image](https://github.com/Kidd254/Math-Magicians-React-App/assets/77106962/4e7b779e-b129-49a9-bb0d-33c774fcfd2d)

#### Hint: Note that the `'Let's do some math'` is aligned to the left of the calculator and not on top of it. 👍 😎